systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -297,6 +300,73 @@ public KafkaBuilder defaultKafka(String name, int kafkaReplicas) {
                     .endSpec();
     }
 
+    DoneableKafka kafkaJBOD(String name, int kafkaReplicas, int diskCount, int diskSizeGi, boolean deleteClaim) {
+        return kafka(jbodKafka(name, kafkaReplicas, diskCount, diskSizeGi, deleteClaim).build());
+    }
+
+    public KafkaBuilder jbodKafka(String name, int kafkaReplicas, int diskCount, int diskSize, boolean deleteClaim) {
```

Comment:
so it's not possible to specify different `deleteClaim` for different disks? We are not going to test it? For example, configuring a JBOD with two disks, one with `deleteClaim` true and the other false and check that only the first PVC will be deleted after cluster deprovisioning.

Reply:
So we can pass disk information in a map where we can store all necessary information and loop over it during creation.

Reply:
@ppatierno I'm planning to add more tests after this one pass code review. I'll do that in another PR.

Reply:
Adding more tests means doing something as suggested by @Frawless so changing the signature of this method. Why do you want to postpone this change?